tests/tests/index_config.rs

```diff
@@ -751,3 +751,396 @@ fn uuid_as_raw_issue2199(mut conn: PgConnection) {
             .fetch_one::<(i64,)>(&mut conn);
     assert_eq!(count, 1);
 }
+
```

Comment:
I think that it would be good to dedupe the harnesses/fixtures for these three tests: I know that opinions are mixed on whether the entire test body should be inlined, but I'm strongly in favor of fixtures in order to make it more obvious which parts of a test are actually unique relative to one another.

Reply:
Good idea. Fixed in [3ed0a79](https://github.com/paradedb/paradedb/pull/2440/commits/3ed0a79f5cca44dea837b3cb08809901419cb709)